docs/1.2. Behaviour.md

```diff
@@ -8,15 +8,19 @@ The Sink Metadata Processing API provides a mechanism to change the settings ass
 
 The initial state of Media Profiles of any Sender is empty. Creating a connection with such a Sender via IS-05 doesn't involve the Sink Metadata Processing API.
 
-A `GET` request returns the last successfully applied Media Profiles array.
+- A `GET` request returns the last successfully applied Media Profiles array.
+- A `PUT` request MUST be validated and if the Device supports the proposed media profile, it initiates an attempt to reconfigure the Device and update corresponding Flow, Source, and/or Sender properties. If successful, the operation MUST return the accepted Media Profiles array, otherwise an error MUST be returned and nothing changed.
+- A `DELETE` request MUST clear the last successfully applied Media Profiles array.
 
-A `PUT` request MUST be validated and if the Device supports the proposed media profile, it initiates an attempt to reconfigure the Device and update corresponding Flow, Source, and/or Sender properties. If successful, the operation MUST return the accepted Media Profiles array, otherwise an error MUST be returned and nothing changed.
+When the `/media-profiles` structure changes as a consequence of PUT or DELETE (request is validated and accepted by the API server), then the associated IS-04 Sender MUST have its version updated (in registered mode this MUST update the registered resource).
 
-A `DELETE` request MUST clear the last successfully applied Media Profiles array.
+If the new `/media-profiles` structure causes underlying IS-04 flows and sources to change, then their version MUST be updated (in registered mode this MUST update the registered resources).
 
-When the `media-profiles` structure changes as a consequence of PUT or DELETE (request is validated and accepted by the API server), then the associated IS-04 Sender MUST have its version updated (in registered mode this MUST update the registered resource).
-
-If the new `media-profiles` structure causes underlying IS-04 flows and sources to change, then their version MUST be updated (in registered mode this MUST update the registered resources).
+When an external change (i.e. not from the `/media-profiles` endpoint) modifies the associated Source and/or Flow, the Sender MUST respect its
+[Media Profiles](1.0.%20Overview.md#media-profile). If the change does not satisfy its Media Profiles the Sender MUST NOT transmit the signal over the network.
```

Comment:
```suggestion
When an external change (i.e. not from the `/media-profiles` endpoint) modifies a Source and/or a Flow of a Sender and/or the Sender itself, it MUST respect [Media Profiles](1.0.%20Overview.md#media-profile) of this Sender. If the change does not satisfy its Media Profiles the Sender MUST NOT transmit the signal over the network.
```

Reply:
Maybe just "any change" not simply those covered currently?

```suggestion
When an external change (i.e. not from the `/media-profiles` endpoint) modifies any aspect of the stream then the Sender MUST respect its
[Media Profiles](1.0.%20Overview.md#media-profile). If the change does not satisfy its Media Profiles the Sender MUST NOT transmit the signal over the network.
```

Reply:
Sounds appropriate. The only thing which seems possibly wrong is that "Sender MUST respect Media Profiles". We can say the change must respect them or use passive voice.